src/upload/system/library/transbank/utils/TransbankSdkWebpay.php

```diff
@@ -39,15 +40,11 @@ public function initTransaction($amount, $sessionId, $buyOrder, $returnUrl)
                     "url" => $response->url,
                     "token_ws" => $response->token
                 ];
-            } else {
-                throw new \Exception('No se ha creado la transacción para, amount: ' . $amount . ', sessionId: ' . $sessionId . ', buyOrder: ' . $buyOrder);
             }
-        } catch (\Exception $e) {
-            $result = [
-                "error" => 'Error al crear la transacción',
-                "detail" => $e->getMessage()
-            ];
-            $this->log->logError(json_encode($result));
+        }  catch (TransactionCreateException $e) {
+
+            $this->log->logError($e->getMessage());
+            return ["error" => $e->getMessage()];
         }
 
         return $result;
```

Comment:
there is a flow where can return $result as an empty array now, is this ok?

Reply:
The code will be modified so that $result does not return empty.